python/sandboxai/integrations/crewai.py

```diff
@@ -0,0 +1,46 @@
+from crewai.tools import BaseTool
+from typing import Type
+from pydantic import BaseModel, Field
+from sandboxai import Sandbox
+
+
+class RunIPythonCellArgs(BaseModel):
+    code: str = Field(..., description="The code to execute in the ipython cell.")
+
+
+class RunIPythonCell(BaseTool):
```

Comment:
Whats the convention on these classes, to call then a verb like Run...?

PS: I think we should add a comment about expected shutdown behavior (on python shutdown)

Reply:
https://github.com/crewAIInc/crewAI-tools/tree/main/crewai_tools/tools

Reply:
Thinking maybe `IPythonTool` or `IPythonSandbox` or `IPythonSandboxTool`

Reply:
Looks like `Tool` as a suffix is a convention

Reply:
Looks like the service as a prefix is common: https://github.com/crewAIInc/crewAI-tools/tree/main/crewai_tools/tools/browserbase_load_tool

Reply:
Leads me to think `SandboxIPythonTool` and `SandboxShellTool`

Reply:
I took this naming scheme from their example but you're right, their own tools use a different naming scheme.

Reply:
or ``SandboxedIPythonTool`